commons/src/main/kotlin/com/valartech/commons/network/blog/HTTPResult.kt

```diff
@@ -58,6 +60,10 @@ interface ErrorResult {
     val exception: Throwable
 }
 
+interface CareCluesError {
```

Comment:
If you need to add in something specific, copy these classes over to the app repo and modify them